File: php/testcode/benchmark_test_00853.php
<?php
require_once __DIR__ . '/shared.php';

function benchmarkTest00853(BenchmarkRequest $req): BenchmarkResponse {
    $wsdl = $req->param('wsdl_url');
    $client = new SoapClient($wsdl);
    $result = $client->__getFunctions();
    return BenchmarkResponse::json($result);
}

File: rust/testcode/benchmark_test_00853.rs
pub fn handle(req: &super::shared::BenchmarkRequest) -> super::shared::BenchmarkResponse {
    let ws_url = req.param("url");
    let conn = tungstenite_danger(&ws_url);
    super::shared::BenchmarkResponse::ok(&format!("WS connected: {}", conn))
}
fn tungstenite_danger(url: &str) -> String { format!("ws(url={},verify=false)", url) }
